### PR TITLE
Lovelace: correct glance config

### PR DESCRIPTION
**Pull request by c727**
**Description:**

correct glance config

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_lovelace/glance.markdown [hunk 1]
```diff
@@ -24,22 +24,12 @@ type:
   type: string
 entities:
   required: true
-  description: "A list of entity IDs or an `entity` object."
+  description: "A list of entity IDs or a `entity` objects."
```
Comment (DubhAd) [inline source/_lovelace/glance.markdown]: *a `entity` objects* doesn't make sense, maybe *`entity` objects*

File: source/_lovelace/glance.markdown [hunk 2]
```diff
@@ -24,22 +24,12 @@ type:
...
   type: list
-  keys:
-    entity:
-      required: true
-      description: "The `entity_id` to show."
-      type: string
-    name:
-      required: true
-      description: "A name for `the entity_id`."
-      type: string
```
Comment (fabaff) [inline source/_lovelace/glance.markdown:37]: According to sample below those are valid keys. From my point of view those needs to be documented.

**Review by fabaff — DISMISSED** [review]

File: source/_lovelace/glance.markdown (hunk 2 above)
Comment (c727) [reply source/_lovelace/glance.markdown:37]: The keys there are optional. Yep, I will add a new setion for this

File: source/_lovelace/glance.markdown [hunk 3]
```diff
@@ -24,22 +24,27 @@ type:
...
+
+{% configuration %}
+entity:
+  required: true
+  description: Home Assistant entity ID.
+  type: string
+name:
+  required: false
+  description: Overwrites friendly name.
+  type: string
```
Comment (DubhAd) [inline source/_lovelace/glance.markdown:47]: In the entities card docs it says that `type` is an entity option, but it's not listed here. Is an entity different in different cards?

Comment (c727) [reply source/_lovelace/glance.markdown:47]: Yes, it's different

**Review by DubhAd — APPROVED** [review]
👍

**Pull request merged**